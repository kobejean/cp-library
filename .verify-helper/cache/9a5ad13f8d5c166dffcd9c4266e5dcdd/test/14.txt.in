2
583 848
87 447
