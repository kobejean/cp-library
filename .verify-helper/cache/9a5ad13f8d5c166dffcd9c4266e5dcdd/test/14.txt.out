318.911037751909740928
